float64[6] jointstate
